src/test/java/teammates/test/cases/webapi/GetFeedbackQuestionRecipientsActionTest.java

```diff
@@ -20,15 +51,232 @@ protected String getRequestMethod() {
         return GET;
     }
 
-    @Test
     @Override
     protected void testExecute() throws Exception {
-        // TODO
+        // Method is required to implement. This comment is used to pass lint
+    }
+
+    @Test
+    public void testExecute_invalidIntend_shouldFail() {
+        ______TS("Invalid intend, should fail");
+        String[] invalidIntendParams = generateParameters(firstSessionInCourse1, 1, Intent.FULL_DETAIL, "", "", "");
+        verifyHttpParameterFailure(invalidIntendParams);
+    }
+
+    @Test
+    public void testExecute_moderatedAndPreviewPersonToGetRecipients_shouldReturnSameRecipientsGotFromGiver() {
+
+        loginAsStudent(student1InCourse1.googleId);
+        String[] questionParams = generateParameters(firstSessionInCourse1, 2, Intent.STUDENT_SUBMISSION, "", "", "");
+        FeedbackQuestionRecipientsData expectedRecipients = getRecipients(questionParams);
+
+        ______TS("Test moderated person get recipient, should be same as recipients got from giver");
+        loginAsInstructor(instructor1OfCourse1.googleId);
+        String[] moderateQuestionParams =
+                generateParameters(firstSessionInCourse1, 2, Intent.STUDENT_SUBMISSION, "", student1InCourse1.email, "");
+        FeedbackQuestionRecipientsData moderatedRecipients = getRecipients(moderateQuestionParams);
+        verifyFeedbackQuestionRecipientsDataEquals(expectedRecipients, moderatedRecipients);
+
+        ______TS("Test preview person get recipient, should be same as recipients got from giver");
+        String[] previewQuestionParams =
+                generateParameters(firstSessionInCourse1, 2, Intent.STUDENT_SUBMISSION, "", "", student1InCourse1.email);
+        FeedbackQuestionRecipientsData previewRecipients = getRecipients(previewQuestionParams);
+        verifyFeedbackQuestionRecipientsDataEquals(expectedRecipients, previewRecipients);
+    }
+
+    @Test
+    public void testExecute_differentRecipientTypes_shouldReturnRecipientsCorrectly() {
```

Comment:
Thanks for your efforts for writing test cases for different types of recipients. However, I should mention that Action-level test is integration test. The logic of providing different recipients for different recipients type is in side `feedbackQuestionsLogic`. So technically, these tests should be inside tests cases for that method.

You don't have to do any change for this. Just let you know and do take note in the future.